js/app.js

```diff
@@ -1,80 +1,131 @@
+import { html, render } from 'lit-html';
+import { repeat } from 'lit-html/directives/repeat.js';
+
+/**
+ * Main app class.
+ *
+ * @class ToDoApp
+ */
 class ToDoApp {
+  /**
+   * Creates an instance of ToDoApp and renders all the pre-exisitng todos.
+   * @memberof ToDoApp
+   */
   constructor() {
-    this.todos = this.getTodos();
+    this.todos = this.getTodosFromLocalStorage();
```

Comment:
`this.todoInput = document.querySelector('#todoInput');`

After we set up polymer we can directly import as a component.
like `import '../component_name`